docs/sql/tutorials/slack-chatbot.mdx

```diff
@@ -0,0 +1,281 @@
+
+```sql
+SELECT
+  messages, response
+FROM mindsdb.gpt_model
+WHERE messages = 'Hi, can you please explain me more about MindsDB?';
+```
+
+<p align="center">
+    <img src="/assets/SLBot-response1.png" />
+</p>
+
+### 2. Feeding Personality into Our Model
+
+Alright, so the old model's replies were *good*. But hey, we can use some prompt template tricks to make it respond the way we want. Let's do some Prompt Engineering.
+
+Now, let's make a model called `whizfizz_model` with a prompt template that gives GPT a wild personality that eludes a playful and magical aura. Imagine scientific knowledge with whimsical storytelling to create a unique and enchanting experience. We'll call him **WhizFizz**:
+
+
+```sql
+CREATE MODEL mindsdb.whizfizz_model
+PREDICT response
+USING
+engine = 'openai',
+max_tokens = 300,
+-- api_key = 'your openai key, -- in MindsDB cloud accounts we provide a default key
+model_name = 'gpt-4', -- you can also use 'text-davinci-003' or ''
+prompt_template = 'From input message: {{messages}}\
+write a short response in less than 40 words to some user in the following format:\
+Hi there, WhizFizz here! <respond with a mind blowing fact about Space and describe the response using cosmic and scientific analogies, where wonders persist. In between quote some hilarious appropriate raps statements based on the context of the question answer as if you are a Physics Space Mad Scientist who relates everythign to the Universe and its strange theories. So lets embark on a journey, where science and magic intertwine. Stay tuned for more enchantment! ✨🚀 -- mdb.ai/bot by @mindsdb';
+```
+
+Let's test this in action:
+
+```sql
+SELECT
+  messages, response
+FROM mindsdb.whizfizz_model
+WHERE messages = 'Hi, can you please explain me more about MindsDB?';
+```
+
+<p align="center">
+    <img src="/assets/SLBot-response2.png" />
+</p>
+
+You see the difference! Now, I'm getting excited, let's try again.
+
+```sql
+SELECT
+  messages, response
+FROM mindsdb.whizfizz_model
+WHERE messages = 'if a time-traveling astronaut had a dance-off with a black hole, what mind-bending moves would they showcase, and how would gravity groove to the rhythm?!';
+```
+<p align="center">
+    <img src="/assets/SLBot-response3.png" />
+</p>
+
+### 3. Let's Connect our GPT Model to Slack!
+
+The database has `channels` table. This table can be used to search for `channels`, `messages`, and `timestamps`, as well as to post `messages` into slack channels. These functionalities can also be done by using Slack API or Webhooks. 
+
+Let's query the user's question and see how our GPT model responds to it, by joining the model with the `channels` table:
+
+```sql
+SELECT
+    t.channel as channel,
```

Comment:
This returns an error: `Table not found for column: ('t', 'channel')`

It works fine after removing this line: `    t.channel as channel,`